README.md

```diff
@@ -1,33 +1,67 @@
 # Create Action
 
-## Objectives
+So we have a form that can accept data and then print out the the values to the screen, that's great for learning how forms work in Rails, however in a real life application we'll most likely want to add the data as a new record in the database. In the `CRUD` lifecycle, this is where the `C` comes in and we `create` a new record.
 
-1. Use the console to create and persist an AR instance
-2. Build a functional create action that can use form data in params to persist a corresponding AR instance
-3. Instantiate an empty instance of a model in create
-4. Manually assign data to an instance from params
-5. Save an instance of AR in a controller action
-6. Understand the missing template error in a create action
-7. Use redirect_to to redirect in a controller using a route helper (redirect_to post_path(@post))
+Before implementing this functionality, let's first open up a Rails console session and create a record manually:
```

Comment:
Include the rails app you want them to be able to run this code in, in this repo